servers/src/grin/server.rs

```diff
@@ -489,14 +491,45 @@ impl Server {
 			.into_iter()
 			.map(|p| PeerStats::from_peer(&p))
 			.collect();
+
+		let tx_stats = TxStats {
+			tx_pool_size: self.tx_pool.read().txpool.entries.len(),
+			stem_pool_size: self.tx_pool.read().stempool.entries.len(),
+		};
+
+		let head = self.chain.head_header().unwrap();
+		let head_stats = ChainStats {
+			last_block_time: head.timestamp,
+			height: head.height,
+			last_block_h: head.prev_hash,
+			total_difficulty: head.total_difficulty(),
+		};
+
+		let disk_usage_gb = {
```

Comment:
I see this is measuring disk usage for the main lmdb database.
We also use significant storage for the various MMR files as well (and the peers lmdb database, but less so).

```
chain_data/lmdb
chain_data/peer
chain_data/header/header_head
chain_data/header/sync_head
chain_data/txhashset/kernel
chain_data/txhashset/output
chain_data/txhashset/rangeproof
```

Reply:
These looked very small which is why I left them out. But I realise now I'm not running full archive and network activity is only low at the moment. I'll see about adding them to the usage statistic.

Reply:
The kernel and rangeproof MMRs should be approx 150MB each on mainnet, regardless of archive mode.

Reply:
Ok I was not looking hard enough.. I see them now. I added the Walkdir crate to go through the whole db_root dir, it's small has a nice API, do you think it's ok?

Reply:
Dependency on walkdir is fine I think. We already use it in the util crate. :+1: